server/mail/mail.go

```diff
@@ -171,19 +171,25 @@ func (m mailService) sendMail(e fleet.Email, msg []byte) error {
 		return nil
 	}
 
-	client, err := dialTimeout(smtpHost)
+	tlsConfig := &tls.Config{
+		ServerName:         e.SMTPSettings.SMTPServer,
+		InsecureSkipVerify: !e.SMTPSettings.SMTPVerifySSLCerts,
+	}
+
+	var client *smtp.Client
+	if e.SMTPSettings.SMTPEnableTLS {
```

Comment:
What's the difference between `SMTPEnableTLS` and `SMTPEnableStartTLS` settings?

Reply:
StartTLS is a protocol command, separate from the TLS protocol: https://sendgrid.com/en-us/blog/what-is-starttls.  In this context it's run if the specified SMTP server supports it.